.github/workflows/industrial_ci_action.yaml

```
@@ -19,6 +13,10 @@ jobs:
         env:
           - {ROS_DISTRO: kinetic, ROS_REPO: main}
           - {ROS_DISTRO: kinetic, ROS_REPO: testing}
+          - {ROS_DISTRO: melodic, ROS_REPO: main}
+          - {ROS_DISTRO: melodic, ROS_REPO: testing}
+          - {ROS_DISTRO: noetic, ROS_REPO: main}
+          - {ROS_DISTRO: noetic, ROS_REPO: testing}
```

I suggested this set of tasks. To limit computation time, I usually limit `ros-testing` to the latest (= actively developed) ROS release.
```suggestion
          - {ROS_DISTRO: kinetic, ROS_REPO: main}
          - {ROS_DISTRO: melodic, ROS_REPO: main}
          - {ROS_DISTRO: noetic, ROS_REPO: main}
          - {ROS_DISTRO: noetic, ROS_REPO: testing}
```